### PR TITLE
[MDS-381] Hover colours update

### PR DESCRIPTION
**Pull request by tkullisaar**

### CODE REVIEW

File: assets/ds-moon-preset.js [hunk 1]
```diff
@@ -319,6 +319,15 @@ module.exports = {
...
+          DEFAULT: withOpacityValue("--jiren"),
+        },
+        "heles": {
+          DEFAULT: withOpacityValue("--heles"),
+        },
+        "zeno": {
+          DEFAULT: withOpacityValue("--zeno"),
+        },
         black: "#000",
         hover: "rgba(34, 34, 37, 0.12)",
```
Comment (phgrey) [inline assets/ds-moon-preset.js:332]: as far as i remeber we wanted to remove these two lines - hover & primary-hover. their usages (only one as i remeber) should be removed also

File: assets/ds-moon-preset.js [hunk 2]
```diff
@@ -319,6 +319,15 @@ module.exports = {
...
         },
+        "jiren": {
+          DEFAULT: withOpacityValue("--jiren"),
+        },
+        "heles": {
+          DEFAULT: withOpacityValue("--heles"),
+        },
+        "zeno": {
+          DEFAULT: withOpacityValue("--zeno"),
+        },
```
Comment (Kypsis) [inline assets/ds-moon-preset.js:330]: Do the `hover` and `primaryHover` under here have use-cases anymore? If so might be better to alias them to the new tokens.

**Review by phgrey — CHANGES_REQUESTED** [review]
also please remove obsolette hover & primary-hover bg classes and their usages

File: assets/ds-moon-preset.js (hunk 2 above)
Comment (phgrey) [reply assets/ds-moon-preset.js:330]: we're not going to use them anymore i believe

Comment (tkullisaar) [reply assets/ds-moon-preset.js:330]: I decided to leave those untouched for the time being, as primary-hover is in use in Button v1 and "hover" parameter as bg-hover in Dropdown, Progress_linear and Single_item_select v1. 
Should I still remove them?

Comment (phgrey) [reply assets/ds-moon-preset.js:330]: no. just mark it as unused anymore and forget please

**Review by phgrey — APPROVED** [review]

File: assets/ds-moon-preset.js (hunk 1 above)
Comment (tkullisaar) [reply assets/ds-moon-preset.js:333]: These legacy classes remain for the time being, with additional explanatory comment.

File: assets/ds-moon-preset.js (hunk 2 above)
Comment (tkullisaar) [reply assets/ds-moon-preset.js:330]: Explanatory comment added.

**Pull request merged**